scripts/start-database.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Entrypoint for running the devserver from within Docker. Before running
+# "make devserver", this file runs database setup / initialization scripts if a
+# database has not already been created.
+
+
+set -e
+
+. /venv/bin/activate
+
+export PATH=$PATH:/venv/bin/
+export PYTHONPATH=$PYTHONPATH:/app
+
+apt-get update && apt-get install -y git
```

Comment:
Should we move this line to the dockerfile?

Reply:
This is another advantage of database setup being in a separate docker container.

It is an optimization to install `git` and dependencies only in the db setup runtime and exit after complete. No cost storage or network cost. This way, the static ambuda docker image remains thinner than with these dependencies installed in the image.